server/plugins/webapp/register-sw.html

```diff
@@ -0,0 +1,7 @@
+<script>
```

Comment:
It could be, but it's such a small file I don't think it makes sense to incur the cost of a network request to load it, so I inlined it.